packages/core/strapi/lib/commands/utils/data-transfer.js

```diff
@@ -341,6 +341,31 @@ const getDiffHandler = (engine, { force, action }) => {
   };
 };
 
+const getAssetsBackupHandler = (engine, { force, action }) => {
+  return async (context, next) => {
+    // if we abort here, we need to actually exit the process because of conflict with inquirer prompt
+    setSignalHandler(async () => {
+      await abortTransfer({ engine, strapi });
+      exitWith(1, exitMessageText(action, true));
+    });
+
+    const confirmed = await confirmMessage(
+      'The backup folder for the assets could not be created inside the public folder. Maybe Strapi does not have write permissions on the public directory. Do you want to continue without the backup?',
+      {
+        force,
+      }
+    );
```

Comment:
```suggestion
    console.warn(
      'The backup for the assets could not be created inside the public directory. Ensure Strapi has write permissions on the public directory.'
    );
    const confirmed = await confirmMessage(
      'Do you want to continue without backing up your public/uploads files?',
      {
        force,
      }
    );
```
Might be nicer on a line by itself, wdyt?